src/js/stores/MesosStream/parsers/ProtobufUtil.js

```diff
@@ -0,0 +1,11 @@
+export function isScalar(value) {
```

Comment:
![image](https://user-images.githubusercontent.com/156010/33212039-85e0ed5c-d121-11e7-94fb-2c0a9183655a.png)

Reply:
-_- not sure if